base/src/main/java/io/fabric8/launcher/base/http/ExternalRequest.java

```diff
@@ -27,19 +27,28 @@ private ExternalRequest() {
         throw new IllegalAccessError("Utility class");
     }
 
+    private static final OkHttpClient client = createClient();
     private static final ObjectMapper mapper = new ObjectMapper();
 
     @Nullable
     public static <T> T execute(Request request, Function<Response, T> consumer) {
-        try (Response response = client.newCall(request).execute()) {
+        try (final Response response = client.newCall(request).execute()) {
             return consumer.apply(response);
         } catch (IOException e) {
             throw new HttpException("Error while executing request", e);
         }
     }
 
+    public static Response execute(Request request) {
+        try (final Response response = client.newCall(request).execute()) {
+            return response;
```

Comment:
-1 Response will be closed.that's why the original method takes a Function

Reply:
ok it makes sense :)